### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by Kyle-Chambers**

### CODE REVIEW

**Comment by Kyle-Chambers** [comment]
screen shot 2018-03-05 at 10 38 54 am

File: client/app/components/questionsAndAnswers.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <p>Page is loading</p>
+      );
+    }
+
+    return (
+      <div className={styles.main}>
+
+        <Header />
+
+        <br></br>
```
Comment (kguinto) [inline client/app/components/questionsAndAnswers.jsx]: Might be better to use padding/margins for whitespace than line breaks

File: tests/test.js [hunk 2]
```diff
@@ -1,49 +1,169 @@
...
+import Question from '../client/app/components/question.jsx';
+import Avatar from '../client/app/components/avatar.jsx';
+import ShowAnswersButton from '../client/app/components/showAnswersButton.jsx';
+import AnswerSubmissionForm from '../client/app/components/answerSubmissionForm.jsx';
+import QuestionSubmissionForm from '../client/app/components/questionSubmissionForm.jsx';
+import Answers from '../client/app/components/answers.jsx';
+import Answer from '../client/app/components/answer.jsx';
+import attractionsData from '../fakeData.js';
+
+import 'isomorphic-fetch'; // / ?????? WTF
```
Comment (kguinto) [inline tests/test.js]: https://github.com/matthew-andrews/isomorphic-fetch ?

File: tests/test.js [hunk 3]
```diff
@@ -1,49 +1,169 @@
...
+import Adapter from 'enzyme-adapter-react-16';
+import QuestionsAndAnswers from '../client/app/components/questionsAndAnswers.jsx';
+import Header from '../client/app/components/header.jsx';
+import Question from '../client/app/components/question.jsx';
+import Avatar from '../client/app/components/avatar.jsx';
+import ShowAnswersButton from '../client/app/components/showAnswersButton.jsx';
+import AnswerSubmissionForm from '../client/app/components/answerSubmissionForm.jsx';
+import QuestionSubmissionForm from '../client/app/components/questionSubmissionForm.jsx';
+import Answers from '../client/app/components/answers.jsx';
+import Answer from '../client/app/components/answer.jsx';
```
Comment (kguinto) [inline tests/test.js:15]: Maybe a stretch goal but you could condense these lines with something like this: https://stackoverflow.com/questions/29722270/import-modules-from-files-in-directory

**Review by kguinto — APPROVED** [review]
Commits could be more detailed, but your coding style is getting a lot more clean & readable overall. 👍

**Pull request merged**